mava/components/jax/executing/base.py

```diff
@@ -46,7 +47,18 @@ def on_execution_init_start(self, executor: SystemExecutor) -> None:
         Returns:
             None.
         """
-        executor._interval = self.config.interval  # type: ignore
+
+        if self.config.evaluation_duration != 1 and (
+            self.config.evaluation_interval is None
+        ):
+            raise ValueError(
+                "Missing evaluation interval value: Evaluation duration"
+                + " value was provided without a value for the evaluation interval."
+            )
+
+        if executor.store.is_evaluator:
+            executor.store._evaluation_interval = self.config.evaluation_interval  # type: ignore # noqa: E501
```

Comment:
why is this variable with an underscore?

Reply:
We were following the convention that [was in dev](https://github.com/instadeepai/Mava/blob/9846ebf90722afba7888c448d139503be198e8e5/mava/components/jax/executing/base.py#L49)  where the `inteval` variable was set using an underscore.

Reply:
It is not a major thing but since the variable is being used externally, without accessor/ mutators needed, I do not think it should be marked private with an underscore

Reply:
Is it okay if we leave it as is then?

Reply:
Done!

Reply:
Thanks